addons/binding/org.openhab.binding.rfxcom/ESH-INF/thing/channels.xml

```diff
@@ -170,4 +170,24 @@
 		<state min="0" max="360" step="1" readOnly="true"></state>
 	</channel-type>
 
+	<channel-type id="scene">
+		<item-type>Number</item-type>
+		<label>Scene</label>
+		<description>Scene channel</description>
+		<state readOnly="false">
+			<options>
+				<option value="0">Scene 1</option>
```

Comment:
a. Great, I guess we should stick to those rather than trying to put in something more sensible.

b. So is the idea here that if the user adds the scene switch to their dashboard, they can set it up so that scene1 is off and scene2 is on (or similar), so that they can have a reliable on/off rather than a toggle? You seem to be adding an abstraction layer on top of the scene commands, which could also be achieved using an abstraction layer in the rules (haven't tested the rules):

Switch SceneSwitch

rule "SceneOn"
when
Item SceneSwitch received command ON
then
livolo.send(SCENE1)
end

rule "SceneOff"
when
Item SceneSwitch received command OFF
then
livolo.send(SCENE2)
end

Would that achieve the same goal, or can we do more by adding it to the code? If the java version doesn't add anything above the rules, then I would recommend taking it out of the java for ease of maintenance.